teammapper-frontend/src/app/core/services/map-sync/map-sync.service.ts

```diff
@@ -429,6 +436,105 @@ export class MapSyncService implements OnDestroy {
       this.mmpService.new(result.map.data, false);
     });
 
+    this.socket.on('mapChangesUndoRedo', (result: ResponseUndoRedoChanges) => {
+      if (result.clientId === this.socket.id) return;
+
+      const ReversePropertyMapping = {
```

Comment:
this should be extracted in a type / constant file. I don't see a reason this has to randomly appear in a function?